### PR TITLE
Simplify logger definition

### PR DESCRIPTION
**Pull request by leplatrem**
@glasserc r?

### CODE REVIEW

File: kinto/config/kinto.tpl [hunk 1]
```diff
@@ -123,7 +123,7 @@ kinto.includes = kinto.plugins.default_bucket
 # kinto.logging_renderer = kinto.core.logs.ClassicLogRenderer
 
 [loggers]
-keys = root, kinto_core
+keys = root
```
Comment (Natim) [inline kinto/config/kinto.tpl]: I think you should rename kinto_core by kinto and log all kinto errors here.

Comment (Natim) [inline kinto/config/kinto.tpl]: Here we lose the fact that we may want the DEBUG log level for kinto, while the INFO level for other package is fine.

**Comment by glasserc** [comment]
I agree with @Natim :)

**Comment by leplatrem** [comment]
Final approval ?

**Comment by glasserc** [comment]
r+

**Pull request merged**